src/main/java/by/exadel/internship/entity/Form.java

```diff
@@ -61,8 +64,21 @@
     @Column(name = "fm_primary_skill")
     private String primarySkill;
 
-//    private InterviewDTO interview;
-//    private FormStatus formStatus;
-//    //    this field is questionable
-//    private TimeForCallDTO timeForCall;
+    @Column(name = "fm_english_level")
+    @Enumerated(value = EnumType.STRING)
+    @Type(type = "by.exadel.internship.mapper.EnumTypePostgreSQL")
+    private EnglishLevel englishLevel;
+
+    @OneToOne
+    @JoinColumn(name = "fm_form_status_id")
+    private FormStatus formStatus;
```

Comment:
As far as i remember formStatus shouldn't be separate entity

Reply:
Done